src/onediff/infer_compiler/utils/env_var.py

```diff
@@ -11,3 +11,7 @@ def parse_boolean_from_env(env_var, default_value):
 
 def set_boolean_env_var(env_var: str, val: bool):
     os.environ[env_var] = str(val)
+
+def set_default_env(env_var, default_value):
+    if env_var not in os.environ:
+        os.environ[env_var] = default_value
```

Comment:
you can use `os.environ.setdefault(env_var, default_value)`

Reply:
> you can use `os.environ.setdefault(env_var, default_value)`

Fixed, thanks for you comment!